egs/wsj/s5/steps/cleanup/internal/resolve_ctm_edits_overlaps.py

```diff
@@ -144,10 +144,14 @@ def choose_best_ctm_lines(first_lines, second_lines,
     """Returns ctm lines that have lower WER. If the WER is the lines with
     the higher number of words is returned.
     """
-    i, best_lines = min((0, first_lines), (1, second_lines),
-                        key=lambda x: wer(x[1]))
-
-    return i
+    if not first_lines:
+        return 1
+    elif not second_lines:
+        return 0
+    else:
+        i, best_lines = min((0, first_lines), (1, second_lines),
```

Comment:
I think if line 139 is modified as:
`return float(num_incorrect_words) / num_words`, this issue would not happen. Maybe you could test that instead of this change here.

Reply:
Yes, this has helped. It's definitely cleaner solution.